website/docs/api/docusaurus.config.js.md

```diff
@@ -80,6 +80,20 @@ module.exports = {
 
 ## Optional fields
 
+### `noIndex`
+
+- Type: `boolean`
+
+Boolean. If true, Docusaurus will politely ask crawlers and search engines to avoid indexing your site. This is done with a header tag and so only applies to docs and pages. Will not attempt to hide static resources. This is a best effort request. Malicious crawlers can and will still index your site.
```

Comment:
This message (backported from v1) looks quite frightening to me 😅 

What about having something simpler in v2, like:

> this option adds `<meta name="robots" content="noindex">` in pages, to tell search engines to avoid indexing the page (more informations [here](https://moz.com/learn/seo/robots-meta-directives))

Reply:
Done